Frontend/CO.CDP.OrganisationApp/Pages/Exclusions/DeclaringExclusions.cshtml

```diff
@@ -0,0 +1,54 @@
+@page "/organisation/{organisationId:guid}/forms/{formId:guid}/sections/{sectionId:guid}/declaring-exclusions"
+
+@model CO.CDP.OrganisationApp.Pages.Exclusions.DeclaringExclusionsModel
+@{
+    var hasError = ((TagBuilder)Html.ValidationMessageFor(m => m.YesNoInput)).HasInnerHtml;
+    var backUrl = $"/organisation/{Model.OrganisationId}/forms/{Model.FormId}/sections/{Model.SectionId}/add-exclusion-declaration";
+}
+<a href="@backUrl" class="govuk-back-link">Back</a>
+
+<main class="govuk-main-wrapper">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <partial name="_ErrorSummary" model="@ModelState" />
+
+            <form method="post">
+                <div class="govuk-form-group @(hasError ? "govuk-form-group--error" : "")">
+                    <fieldset class="govuk-fieldset">
+                        <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+                            <h1 class="govuk-fieldset__heading">
+                                Do you have any exclusions to add for your organisation or a connected person?
+                            </h1>
+                        </legend>
+                        @if (hasError)
+                        {
+                            <p class="govuk-error-message">
+                                <span class="govuk-visually-hidden">Error:</span>
+                                @Html.ValidationMessageFor(m => m.YesNoInput)
+                            </p>
+                        }
+
+                        <div class="govuk-radios" data-module="govuk-radios">
+                            <div class="govuk-radios__item">
+                                <input class="govuk-radios__input" id="conditional-YES" name="YesNoInput" type="radio"
+                                       value="true" data-aria-controls="conditional-YES" @(Model.YesNoInput == true ? "checked" : "")>
+                                <label class="govuk-label govuk-radios__label" for="conditional-YES">Yes</label>
+                            </div>
+                            <div class="govuk-radios__item">
+                                <input class="govuk-radios__input" id="conditional-NO" name="YesNoInput" type="radio"
+                                       value="false" data-aria-controls="conditional-NO" @(Model.YesNoInput == false ? "checked" : "")>
```

Comment:
`data-aria-controls` attributes are used by the govuk JavaScript to control conditionally revealing content (See https://github.com/alphagov/govuk-frontend/blob/main/packages/govuk-frontend/src/govuk/components/radios/radios.mjs#L21)

That doesn't seem to be the case here so these could be removed from both inputs

Reply:
Thank you Andy - thats a very good point. I have removed it.